manifests/service.pp

```diff
@@ -2,22 +2,37 @@
 # @api private
 class pulpcore::service {
 
+  ['gunicorn', 'rq'].each |$bin| {
+    file { "/usr/libexec/pulpcore/${bin}":
+      ensure  => file,
+      content => "#!/bin/bash\nexec ${bin} \"$@\"\n",
+      owner   => 'root',
+      group   => 'root',
+      mode    => '0755',
```

Comment:
Is worldable executable applicable in our case?

Reply:
It doesn't hurt since there's nothing secret in there.

Reply:
Does it imply any other user could run those commands? Starting any rogue services?

Reply:
Yes, but they can already do that today using `/usr/bin/gunicorn` so there is no difference. I have a suggestion to use unix sockets instead of TCP ports but the current SELinux policy in Pulpcore doesn't allow that now (https://pulp.plan.io/issues/7189). By using systemd socket activation and letting systemd create `/run/pulpcore-api.sock` (or similar) owned by `apache:root` and mode `0600` we can prevent any local user from spinning up a rogue service. It also prevents local users from impersonating admin by connecting directly to the TCP port.